cinema/models.py

```diff
@@ -1,10 +1,33 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class Genre(models.Model):
+    name = models.CharField(unique=True, max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movie")
+    genres = models.ManyToManyField(Genre, related_name="movie")
```

Comment:
Related name shoud be in a plural form, so that you would call the queryset with `Actor.movies`, not `Actor.movie`. Rewrite the related name, please